index.js

```diff
@@ -48,6 +48,53 @@ export class PProgress extends Promise {
 		});
 	}
 
+	static allSettled(promises, options) {
```

Comment:
```suggestion
	static allSettled(promises, {concurrency} = {}) {
```

Better to be explicit about what we include.